graphql/resolvers.go

```diff
@@ -290,24 +281,31 @@ func (r *queryResolver) Hosts(ctx context.Context, hostID *string, distroID *str
 	}, nil
 }
 
-func (r *queryResolver) MyHosts(ctx context.Context) ([]*restModel.APIHost, error) {
```

Comment:
I think you accidentally removed these from a bad merge. we should keep these

Reply:
sorry, mistake. fixed.